### PR TITLE
add MODIFY COLUMN compression codec section, sync RU and EN formatting and links

### PR DESCRIPTION
**Pull request by Slach**
Signed-off-by: Slach <bloodjazman@gmail.com>
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/ru/sql-reference/statements/alter/column.md [hunk 1]
```diff
@@ -189,4 +220,3 @@ ALTER TABLE table_with_ttl RENAME COLUMN column_ttl TO column_ttl_new;
 Запрос `ALTER` блокирует все чтения и записи для таблицы. То есть, если на момент запроса `ALTER`, выполнялся долгий `SELECT`, то запрос `ALTER` сначала дождётся его выполнения. И в это время, все новые запросы к той же таблице, будут ждать, пока завершится этот `ALTER`.
```
Comment (mathalex) [inline docs/ru/sql-reference/statements/alter/column.md:220]:
```suggestion
Запрос `ALTER` блокирует все чтения и записи для таблицы. То есть если на момент запроса `ALTER` выполнялся долгий `SELECT`, то запрос `ALTER` сначала дождётся его выполнения. И в это время все новые запросы к той же таблице будут ждать, пока завершится этот `ALTER`.
```

**Comment by Slach** [comment]
@mathalex thanks for review

File: docs/en/sql-reference/statements/alter/column.md [hunk 2]
```diff
@@ -20,7 +20,6 @@ The following actions are supported:
 
 -   [ADD COLUMN](#alter_add-column) — Adds a new column to the table.
 -   [DROP COLUMN](#alter_drop-column) — Deletes the column.
--   [RENAME COLUMN](#alter_rename-column) — Renames the column.
```
Comment (mathalex) [inline docs/en/sql-reference/statements/alter/column.md:23]: The file contains two paragraphs with RENAME COLUMN, so I suppose that the second one (at the end) is not needed. You can just simply delete it (lines 196-211) and update this list of actions (move the function title on the third position like it was before).
Same with the Russian version.

Comment (Slach) [reply docs/en/sql-reference/statements/alter/column.md:23]: fixed at https://github.com/ClickHouse/ClickHouse/pull/26878/commits/f9c6df4d645cf9af108b1af1ed37f0a72a3db490

**Review by mathalex — COMMENTED** [review]
I have meant this (two last commits) (to delete the whole repeating paragraph; also to change the order in the list of actions - you have already done that).

**Comment by Slach** [comment]
@mathalex ahh, sorry, I miss about double content for REMOVE  part
so, look like PR ready to merge?

**Review by qoega — APPROVED** [review]

**Pull request merged**